src/bcc_sensor.c

```diff
@@ -10,12 +10,13 @@
 #ifdef randomized_struct_fields_start
 #undef randomized_struct_fields_start
 #endif
-#define randomized_struct_fields_start  struct {
-
+#define randomized_struct_fields_start struct {
 #ifdef randomized_struct_fields_end
 #undef randomized_struct_fields_end
 #endif
-#define randomized_struct_fields_end    };
+#define randomized_struct_fields_end \
```

Comment:
What is the reason for adding the extra whitespace here?

Reply:
It looks like he ran clang-format on the source for the first commit.

Reply:
I think that's noise from clang-format 🤷